models.go

```diff
@@ -251,6 +252,19 @@ type Nip47PayResponse struct {
 	Preimage string `json:"preimage"`
 }
 
+type Nip47MultiPayParams struct {
+	Invoices []InvoiceInfo `json:"invoices"`
+}
+
+type InvoiceInfo struct {
+	Id      uint64 `json:"id"`
```

Comment:
Shouldn't `id` be a string? and it should be optional.

Also `InvoiceInfo` naming is vague, this is actually specifically for multi_pay_invoice right? the value should be a `Nip47PayParams` with an additional, optional id. I think you can embed `Nip47PayParams` inside the struct e.g.
```
type Nip47MultiPayInvoiceElement {
  Nip47PayParams
  Id *string `json:"id"`
}
```